server/routes/probationPractitionerReferrals/probationPractitionerReferralsController.test.ts

```diff
@@ -415,29 +415,6 @@ describe('GET /probation-practitioner/referrals/:id/details', () => {
         })
     })
   })
-  describe('when no team details can be found', () => {
```

Comment:
Removed these tests since:

1. the if logic has been removed from the njk class
2. Writing a test for empty team details will be messy since will have to include a big chunk of html.
i.e.:

```
 <h2 class=\"govuk-heading-m\">Team contact details</h2>················
<dl class=\"govuk-summary-list\">········
      <div class=\"govuk-summary-list__row\">
        <dt class=\"govuk-summary-list__key\">
          Phone
        </dt>
        <dd class=\"govuk-summary-list__value\">···········
        </dd>·········
      </div>·············
      <div class=\"govuk-summary-list__row\">
        <dt class=\"govuk-summary-list__key\">
          Email address
        </dt>
        <dd class=\"govuk-summary-list__value\">···········
        </dd>·········
      </div>········
</dl>
```

Reply:
Hey Andrew - thanks for this explanatory comment, it's super useful. I think that it would be great to include this in the commit message instead of as a PR comment, so that somebody viewing your commit in the future can also understand the motivation. Take a look at our [team's decision about using Git commits to explain our work](https://structurizr.com/share/56937/decisions/Refer%20and%20monitor%20an%20intervention#3), in particular the section "The Git repository is probably immortal".

Reply:
Ok. I'll be more descriptive in my commits in future.
I have amended the commit message.